documentation/tutorials/AnalyzeOutput/modelVSdata.Rmd

```diff
@@ -31,62 +36,111 @@ Within both the run and out folders there will be one folder for each unique mod
+library(PEcAn.benchmark)
 ```
 
-The arguments to read.output are the run ID, the folder where the run is located, the start year, the end year, and the variables being read.
+## Read in settings
 
-##	Compare model to flux observations
+There are two ways to access runs from pecan. One is from a settings object generated from an xml file, and the other is to query the database using the run ID, site ID, and output files manually. These ID's can all be found in the run README file. Both are shown below.  
 
-**First** _load up the observations_ and take a look at the contents of the file
+**From an XML file**
+```{r}
+## Read in the xml
+settings<-read.setting("filepath/custom.xml")
 
+## To read in the model output 
+runid<-settings$workflow$id
+outdir<- settings$outdir
+start.year<-settings$run$start.date
+end.year<-settings$run$end.date
+
+site_id<-settings$run$site$id
+
+## Open up a connection to The Bety Database 
+bety <- src_postgres(host = settings$database$bety$host, user = settings$database$bety$user, password = settings$database$bety$password, dbname = settings$database$bety$dbname)
+
+```
+
+
+**From ID's**
 ```{r}
-library(PEcAn.assim.batch)
-obs <- read.csv("~/output/dbfiles/AmerifluxLBL_site_0-772/AMF_US-NR1_BASE_HH_9-1.csv",skip=2)
-names(obs)
+
+## To read in the model output 
+runid<- 99000000001
+outdir<- "/fs/data2/output/PEcAn_99000000001/out/99000000001"
+start.year<-1997
+end.year<-2004
+site.id<-853
+
+## Open up a connection to The Bety Database
+bety<-list(host = "psql-pecan.bu.edu", driver = "PostgreSQL", user = "bety", dbname = "bety", password = "bety")
+bety <- src_postgres(host = bety$host, user = bety$user, password = bety$password, dbname = bety$dbname)
 ```
 
-Here 0-772 is the database site ID, a folder that was created when you selected the site and PEcAn downloaded the Ameriflux file from the Ameriflux website. 
 
-**Second**, _change all missing values_ from -9999 to not a number (NA) and also _$u*$ filter_ the fluxes. Here we will apply a conservative $u*$ of 0.2.
 
+## Read in model output from spesific variables
 ```{r}
-obs[obs==-9999]=NA
-obs$NEE_PI[obs$USTAR<0.2]=NA
+model_vars<-c("time", "NEE") #varibles being read
+
+model <- read.output(runid,outdir,start.year, end.year, model_vars,dataframe=TRUE)
 ```
 
-**Third**, _convert the model to have the same units as the data_
+The arguments to read.output are the run ID, the folder where the run is located, the start year, the end year, and the variables being read. The README file in the Input file dropdown menue of any sucessful run lists the run ID, the output folder, and the start and end year. 
+
+##	Compare model to flux observations
+
+**First** _load up the observations_ and take a look at the contents of the file
+
 
 ```{r}
-modNEE <- udunits2::ud.convert(out$NEE,"kg m-2 s-1","ug m-2 s-1")/12.0 #12 g C per mol
+
+File_Path<-"/fs/data3/tmccabe/valicali/AMF_US-Dk3_BASE_HH_3-1.csv" #This is a path to where the observations are stored. 
```

Comment:
this should be something that's more generic -- a path on the demo VM that matches a site they would have just run (e.g. Niwot)